### PR TITLE
docs(android): guides for turn-key setup

### PR DESCRIPTION
**Pull request by space-pope**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

Updates the Android quickstart to use the quickest method possible for...starting. I also added a separate guide for the simplified config, as there's a bit more to talk about than I want to cover in a quickstart.

### CODE REVIEW

File: content/docs/Android/getting-started.md [hunk 1]
```diff
@@ -82,58 +73,86 @@ Note that sending audio over the network can use a considerable amount of data,
...
 
-### 2. `SpeechPipeline`
+### 2. `Spokestack`
 
-With the proper permissions in place, it's time to decide where you'd like to receive and process speech input. In a single-activity app, the easiest place for this is going to be your main activity. `import io.spokestack.spokestack.SpeechPipeline` at the top of the file, and add a `SpeechPipeline` member:
+With the proper permissions in place, it's time to decide where you'd like to receive and process speech input. In a single-activity app, the easiest place for this is going to be your main activity. `import io.spokestack.spokestack.Spokestack` at the top of the file, and add a `Spokestack` member:
 
 ```kotlin
-private var pipeline: SpeechPipeline? = null
+private lateinit var spokestack: Spokestack
```
Comment (noelweichbrodt) [inline content/docs/Android/getting-started.md:81]: `lateinit` 🤦🏻‍♂️

File: content/docs/Android/getting-started.md [hunk 2]
```diff
@@ -82,58 +73,86 @@ Note that sending audio over the network can use a considerable amount of data,
...
+You'll probably want to build the pipeline when the activity is created. Remember that you'll need to have the `RECORD_AUDIO` permission for this, so make sure you check that permission before trying to _start_ Spokestack.
 
 ```kotlin
-pipeline = SpeechPipeline.Builder()
-    .useProfile("io.spokestack.spokestack.profile.VADTriggerAndroidASR")
-    .setAndroidContext(applicationContext)
-    .addOnSpeechEventListener(this)
-    .build();
+spokestack = Spokestack.Builder()
+    .setProperty("wake-detect-path", "$cacheDir/detect.tflite")
```
Comment (noelweichbrodt) [inline content/docs/Android/getting-started.md:88]: do you reference how to obtain the wakeword and nlu files somewhere else earlier?

File: content/docs/Android/getting-started.md [hunk 3]
```diff
@@ -144,66 +163,60 @@ We've listed all possible speech events here; see [the documentation](https://ww
...
+        Log.i(logTag, "\t\t${slot.key}: ${slot.value.value}")
     }
+    respond(result.utterance)
+}
+
+private fun respond(utterance: String) {
+    // A (too-) simple response generator that parrots back what the user just said. With
+    // the default TTS setup, this response will be automatically played when the audio is
+    // available.
+    val request = SynthesisRequest.Builder("Why do you feel that $utterance?").build()
```
Comment (noelweichbrodt) [inline content/docs/Android/getting-started.md:185]: Nice ELIZA reference

File: content/docs/Android/turnkey-configuration.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+```kotlin
+val spokestack = Spokestack.Builder()
+    .setProperty("nlu-model-path", "path-to-nlu.tflite")
+    .setProperty("nlu-metadata-path", "path-to-metadata.json")
+    .setProperty("wordpiece-vocab-path", "path-to-vocab.txt")
+    // ...
+    .build()
+```
+
+Once configured, all ASR transcripts will automatically be sent through NLU; see [Receiving events](receiving-events) below for information on how to see the results.
```
Comment (noelweichbrodt) [inline content/docs/Android/turnkey-configuration.md]: is this an in-document anchor link?

**Review by noelweichbrodt — APPROVED** [review]
LGTM

File: content/docs/Android/getting-started.md (hunk 1 above)
Comment (space-pope) [reply content/docs/Android/getting-started.md:81]: When you want to init...later. A language keyword that says "We want null safety, but not _really_."

File: content/docs/Android/turnkey-configuration.md (hunk 4 above)
Comment (space-pope) [reply content/docs/Android/turnkey-configuration.md]: Yes, but it is an incorrect in-document anchor link. Thanks for catching it.

File: content/docs/Android/getting-started.md (hunk 2 above)
Comment (space-pope) [reply content/docs/Android/getting-started.md:88]: I don't, but I'll add a link to the other document that does. If they don't want to bother with that when going through the quickstart, I'd rather they just keep reading and do the "everything but the neural stuff" setup that comes next.

**Pull request merged**